server/src/services/cleanUserInput.js

```diff
@@ -0,0 +1,8 @@
+export const cleanUserInput = (formInput) => {
```

Comment:
Could you add eslint-ignores to the console.logs in SquidSeeder? The eslint warnings clutter up the PR a little :)